### PR TITLE
Add addition info for Kysely

### PR DESCRIPTION
**Pull request by joan-ing**
Closes #

## Summary

- Set docs page to `true` and Add Alexis code snippet from blog post to docs

---

### Remaining work

The following work still needs to be completed:

- [ ] <List outstanding tasks here>

### Dependencies

- [ ] <Tasks that must be completed before merging this pull request>

---

### Timing

**Release**:

- [x] When ready
- [ ] Hold for release | Release after: <mm/dd/yy>

### CODE REVIEW

File: 070-Integrations/040-kysley.mdx [hunk 1]
```diff
@@ -1,13 +1,67 @@
...
+pnpm add kysely @xata.io/kysely @xata.io/client
+```
+
+```sh
+yarn add kysely @xata.io/kysely @xata.io/client
+```
+
+```sh
+npm install kysely @xata.io/kysely @xata.io/client
+```
```
Comment (SferaDev) [inline 070-Integrations/040-kysley.mdx:32]:
````suggestion
```sh
npm install kysely @xata.io/kysely @xata.io/client
```

```sh
pnpm add kysely @xata.io/kysely @xata.io/client
```

```sh
yarn add kysely @xata.io/kysely @xata.io/client
```
````

File: 070-Integrations/040-kysley.mdx [hunk 2]
```diff
@@ -1,13 +1,67 @@
...
+### Usage 
+
+Kysely is a query builder library for TypeScript that helps in generating SQL queries. For compatibility with Xata, a specialized dialect has been developed for Kysely. The [Xata dialect for Kysely](https://github.com/xataio/client-ts/tree/main/packages/plugin-client-kysely) enables the use of Xata-specific functionality with Kysely.
+
+The following example, demonstrates how you can employ the Xata dialect for Kysely:
+
+```tsx
+import { Kysely } from 'kysely';
+import { XataDialect, Model } from '@xata.io/kysely';
+import { DatabaseSchema, getXataClient } from 'xata';   // generated client
```
Comment (SferaDev) [inline 070-Integrations/040-kysley.mdx:45]:
```suggestion
import { DatabaseSchema, getXataClient } from 'xata'; // Generated client
```

File: 070-Integrations/040-kysley.mdx [hunk 3]
```diff
@@ -1,13 +1,67 @@
...
+
+console.log(drivers);
+```
+
+In the provided code example, the following steps are being taken to integrate Xata with Kysley:
+
+1. Install the Xata client as usual, see our [getting started guide](/docs/getting-started/installation).
+1. Import `getXataClient` from the generated client (usually generated by `xata pull` in `src/xata.ts`).
+1. Instantiate Kysley with the Xata dialect. Note that the `Model` type adapts the Xata generated `DatabaseSchema` type to the `Model` type used by Kysely.
+1. Use Kysley as you would normally do. Type safety and autocompletion should work.
```
Comment (SferaDev) [inline 070-Integrations/040-kysley.mdx:67]:
```suggestion
1. Use Kysley as you would normally do with type safety and autocompletion.

```

File: 070-Integrations/040-kysley.mdx [hunk 4]
```diff
@@ -1,13 +1,67 @@
...
+  .where("nationality", "=", "Spanish")
+  .execute();
+
+console.log(drivers);
+```
+
+In the provided code example, the following steps are being taken to integrate Xata with Kysley:
+
+1. Install the Xata client as usual, see our [getting started guide](/docs/getting-started/installation).
+1. Import `getXataClient` from the generated client (usually generated by `xata pull` in `src/xata.ts`).
```
Comment (SferaDev) [inline 070-Integrations/040-kysley.mdx:65]:
```suggestion
1. Import `getXataClient` from the generated client (usually generated by `xata pull` in a file of your project).
```

File: 070-Integrations/040-kysley.mdx [hunk 5]
```diff
@@ -1,13 +1,67 @@
...
+1. Instantiate Kysley with the Xata dialect. Note that the `Model` type adapts the Xata generated `DatabaseSchema` type to the `Model` type used by Kysely.
```
Comment (SferaDev) [inline 070-Integrations/040-kysley.mdx:66]:
```suggestion
1. Instantiate Kysley with the Xata dialect. Note that the `Model` type adapts the Xata generated `DatabaseSchema` type to a `Model` compatible with Kysely.
```

**Review by SferaDev — APPROVED** [review]

File: 070-Integrations/040-kysley.mdx [hunk 6]
```diff
@@ -1,13 +1,67 @@
 ---
 title: Kysley with Xata
 navTitle: Kysley
-keywords: ['build', 'platform']
-description: Xata integrates with Netlify to automate migrations and support the GitHub workflow for efficient web app deployment
+keywords: ['orm', 'kysley']
```
[the remark this answers is not in the document]
Comment (SferaDev) [reply 070-Integrations/040-kysley.mdx]: Kysely is a query builder, drizzle is an ORM

**Review by tsg — APPROVED** [review]

**Comment by SferaDev** [comment]
@tsg We lack drizzle now 😺

File: 070-Integrations/025-drizzle.mdx [hunk 7]
```diff
@@ -0,0 +1,36 @@
+---
+title: Drizzle with Xata
+navTitle: Drizzle
+keywords: ['orm', 'kysley', 'typescript', 'query builder']
```
Comment (SferaDev) [inline 070-Integrations/025-drizzle.mdx:4]:
```suggestion
keywords: ['orm', 'drizzle', 'typescript', 'query builder']
```

File: 070-Integrations/025-drizzle.mdx [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+description: "Use the Drizzle integration as a TypeScript ORM"
+slug: integrations/drizzle
+published: true
+---
+
+<Alert status="warning">
+  The integration of Drizzle with Xata is currently in an experimental phase and is not yet fully supported.
+</Alert>
+
+[Drizzle](https://github.com/drizzle-team/drizzle-orm) is a TypeScript ORM that is focused on type safety and follows closely follows the SQL syntax.
```
Comment (SferaDev) [inline 070-Integrations/025-drizzle.mdx:14]:
```suggestion
[Drizzle](https://github.com/drizzle-team/drizzle-orm) is a TypeScript ORM that is focused on type safety and closely follows the SQL syntax. 
```

**Review by tsg — APPROVED** [review]

File: 070-Integrations/025-drizzle.mdx [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+});
+
+const db = drizzle(xata);
+
+const result = await db
+  .select({ surname: drivers.surname, forename: drivers.forename })
+  .from(drivers)
+  .where(eq(drivers.nationality, "Spanish"))
+  .execute();
+ ```
```
Comment (SferaDev) [inline 070-Integrations/025-drizzle.mdx:60]:
````suggestion
```tsx
import { pgTable, text } from 'drizzle-orm/pg-core';
import { drizzle } from '@xata.io/drizzle';
import { eq } from 'drizzle-orm';
import { getXataClient } from './xata';

const xata = getXataClient();

const drivers = pgTable('drivers', {
  id: text('id').primaryKey(),
  surname: text('surname'),
  forename: text('forename'),
  nationality: text('nationality')
});

const db = drizzle(xata);

const result = await db
  .select({ surname: drivers.surname, forename: drivers.forename })
  .from(drivers)
  .where(eq(drivers.nationality, 'Spanish'))
  .execute();

```
````

File: 070-Integrations/025-drizzle.mdx [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+
+### Usage 
+
+You can use Drizzle with a Xata, as demonstrated in the following examples:
+
+```tsx
+import { pgTable, text } from 'drizzle-orm/pg-core';
+import { drizzle } from '@xata.io/drizzle';
+import { eq } from 'drizzle-orm';
+import { getXataClient } from './xata';
```
Comment (SferaDev) [inline 070-Integrations/025-drizzle.mdx:42]:
```suggestion
import { getXataClient } from './xata'; // Generated client
```

File: 070-Integrations/025-drizzle.mdx (hunk 9 above)
Comment (tsg) [reply 070-Integrations/025-drizzle.mdx]: That's whitespace only, right?

File: 070-Integrations/025-drizzle.mdx [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+```
+
+```sh
+pnpm add drizzle-orm @xata.io/drizzle @xata.io/client
+```
+
+```sh
+yarn add drizzle-orm @xata.io/drizzle @xata.io/client
+```
+
```
Comment (SferaDev) [inline 070-Integrations/025-drizzle.mdx:33]:
```suggestion
</TabbedCode>

```

File: 070-Integrations/040-kysley.mdx [hunk 12]
```diff
@@ -1,13 +1,65 @@
...
+```
+
+```sh
+pnpm add kysely @xata.io/kysely @xata.io/client
+```
+
+```sh
+yarn add kysely @xata.io/kysely @xata.io/client
+```
+
```
Comment (SferaDev) [inline 070-Integrations/040-kysley.mdx:33]:
````suggestion
```sh
npm install kysely @xata.io/kysely @xata.io/client
```

```sh
pnpm add kysely @xata.io/kysely @xata.io/client
```

```sh
yarn add kysely @xata.io/kysely @xata.io/client
```
````

File: 070-Integrations/025-drizzle.mdx [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+const result = await db
+  .select({ surname: drivers.surname, forename: drivers.forename })
+  .from(drivers)
+  .where(eq(drivers.nationality, 'Spanish'))
+  .execute();
+```
+
+The code imports required modules from the `drizzle-orm` and `@xata.io/drizzle` libraries. It also brings in the `eq` function from `drizzle-orm` for value comparison and the `getXataClient` function from a local xata module. The `getXataClient` function is used to acquire an instance of the Xata client.
+
+Currently, we recommend manually defining the schema like in the example above. We are working on utility functions to automatically generate the Drizzle model from the Xata database schema.
```
Comment (SferaDev) [inline 070-Integrations/025-drizzle.mdx:64]:
```suggestion
Currently, we recommend manually defining the schema like in the example above. We are working on utility functions to automatically generate the Drizzle model from the Xata database schema.

### Limitations

- Drizzle migrations are not supported yet
- Transactions are not supported yet

```

**Pull request merged**